From 3b53dae2ad9c382409af146fa9d3037953452fed Mon Sep 17 00:00:00 2001
From: Tom Schoonjans <Tom.Schoonjans@genomicsplc.com>
Date: Sat, 6 Jun 2026 21:45:23 +0100
Subject: [PATCH] Fix meson bug

---
 python/meson.build | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/python/meson.build b/python/meson.build
index 04f37aa1..131f330c 100644
--- a/python/meson.build
+++ b/python/meson.build
@@ -54,7 +54,7 @@ if swig.found()
         install: true,
         install_dir: python.get_install_dir(pure: false),
         c_args: core_c_args + ['-I' + numpy_header_location] + python_error_flags,
-        install_tag: ['python-runtime']
+        install_tag: 'python-runtime'
     )
     python_enabled = true
 endif
@@ -86,7 +86,7 @@ if cython.found()
         install: true,
         install_dir: python.get_install_dir(pure: false),
         c_args: core_c_args + ['-I' + numpy_header_location] + python_error_flags,
-        install_tag: ['python-runtime'],
+        install_tag: 'python-runtime',
     )
     python_numpy_enabled = true
 endif
